viz/charts/linkerd-viz/values.yaml

```diff
@@ -349,6 +349,12 @@ grafana:
   # dashboards. See the [Linkerd
   # documentation](https://linkerd.io/2/tasks/grafana) for more information
   url:
+  # -- url of a Grafana instance hosted off-cluster. Can not be set if
+  # grafana.url is set. The reverse proxy will not be used for this URL.
```

Comment:
Can you complete the comment above `url` saying also that it can't be set if `externalUrl` is also set? And while you're at it, could you also specify in that same coment that the `url` setting is specific for in-cluster Grafana instances?

Reply:
Yep, comment has been updated.